ql/src/java/org/apache/hadoop/hive/ql/security/authorization/plugin/AuthorizationMetaStoreFilterHook.java

```diff
@@ -46,11 +46,18 @@ public AuthorizationMetaStoreFilterHook(Configuration conf) {
   }
 
   @Override
-  public List<String> filterTableNames(String catName, String dbName, List<String> tableList)
+  public List<String> filterTableNames(String dbName, List<String> tableList)
       throws MetaException {
     List<HivePrivilegeObject> listObjs = getHivePrivObjects(dbName, tableList);
     return getTableNames(getFilteredObjects(listObjs));
   }
+
+  @Override
+  public List<String> filterTableNames(String catName, String dbName, List<String> tableList)
+      throws MetaException {
+    return filterTableNames(dbName, tableList);
```

Comment:
This seems problematic to me.
If we ignore the catalog name, that could become a serious hidden issue.

Do we have a better solution to this?
Maybe throw an exception if the catalog name is not the default? Or do the filtering correctly with the catalog info as well?